docs/guides/operator/installation.adoc

```diff
@@ -21,6 +21,66 @@ The recommended way to install the {project_name} Operator in Kubernetes environ
 
 * Be sure that you have cluster-admin permission or an equivalent level of permissions granted by an administrator.
 
+[WARNING]
```

Comment:
This structure looks a bit weird to me:
<img width="1085" alt="Screenshot 2025-03-27 at 17 07 24" src="https://github.com/user-attachments/assets/2b445512-010a-42ae-8164-d5031b001d73" />
Maybe we could swap the warning box with the "Configuring Manual Approval for OLM Upgrades" heading so it's under that?